### DISCUSSION TITLE
调用ollama里的模型会话时长只有5分钟[Bug]

### DISCUSSION THREAD

**Discussion by Jerryye17** [lobehub/lobehub#2310] [Self Hosting] [answered]
### 💻 系统环境

Other Linux

### 📦 部署环境

Docker

### 🌐 浏览器

Chrome

### 🐛 问题描述

例如LobeChat调用Ollama里的llama3模型，首次提问，ollama会加载llama3模型，一直会话没问题，但一旦不提问后，5分钟模型会被卸载，当再次问问题，又要重新加载llama3模型，对于本地资源不是特别好的情况下，模型加载基本要达到30-60秒，用户体验比较差。
在ollama open webui里，在通用的高级设置里是可以设置保持会话时长的，但LobeChat没有此设置。

### 🚦 期望结果

能在LobeChat的ollama模型里，设置保持会话时长。

[a comment by an automated account is not shown]

**Comment by arvinxx** [comment, marked as answer]
在 LobeChat 里做可能不一定行，你得自己设定 ollama 的环境变量吧？

https://lobehub.com/zh/docs/self-hosting/examples/ollama#ollama-%E7%8E%AF%E5%A2%83%E5%8F%98%E9%87%8F

[a comment by an automated account is not shown]

**Comment by Jerryye17** [comment]
感谢，部署时加参数     -e OLLAMA_KEEP_ALIVE="5m"   可以控制模型保持会话的时长

[a comment by an automated account is not shown]
